dynamiqs/utils/utils.py

```diff
@@ -735,3 +735,24 @@ def mpow(x: Tensor, n: int) -> Tensor:
                 [0.+0.j, 1.+0.j]])
     """
     return torch.linalg.matrix_power(x, n)
+
+def entropy_vn(x: Tensor) -> Tensor:
+    """Von-Neumann entropy of density matrix
+
+    Args:
+        x _(..., n, n)_: Square matrix.
+
+    Returns:
+        _(...)_ Von_Neumann entropy of x
+
+    Examples:
+        >>> rho = 0.5*dq.fock_dm(2,0) + 0.5*dq.fock_dm(2,1)
+        >>> entropy_vn(rho)
+        tensor(0,6931471806)
```

Comment:
```suggestion
        tensor(0.693)
```
(as discussed with you yesterday, the example is tested by `task doctest-code`, and it won't pass until exactly correct).